### DISCUSSION TITLE
🍺🍺 Bumpkin Brew for Event Tickets (Trade Left Over / Remaining To Closeout Event)

### DISCUSSION THREAD

**Discussion by GoliathRulz** [sunflower-land/sunflower-land#2073] [Ideas]
I would like to propose something that is typically done in other games, which is to trade in residual or remaining tickets on an event for experience.  Now that we have that we could trade in remaining tickets for a special brew that is given out by the event organizers Bumpkin Brew.

Bumpkin Brew:

For each ticket of an event given a player can receive 10 experience / brew bottle on their bumpkin by consuming Bumpkin Brew.  Tickets are traded in at the Smoothie Shack and the "free" brew is given out.   This is a consumable and is not tradable like other foods, but can be used to boost your bumpkin. 

Bumpkin Brew could be an option at the Grub Shop as an item to trade in for 2 SLF if that is something that could be considered, and would be one of the random items that are in the Grub Shop.

**Comment by FarmerKuenster** [comment]
I really like this idea!
